rel/config/ee-examples/cinfo-authn.conf

```diff
@@ -0,0 +1,30 @@
+authentication = [
+  {
+    mechanism = cinfo
+    checks = [
+      # allow clients with username starts with 'super-'
+      {
+        is_match = "regex_match(username, 'super-')"
```

Comment:
is_match = "regex_match(username, '^super-')"

Reply:
thank you, fixed.